src/render_engine/cli/cli.py

```diff
@@ -82,6 +84,52 @@ def _create_site_with_vars(
     return site
 
 
+def get_available_themes(console: Console, app: Site, theme_name: str) -> list[str]:
```

Comment:
Please add docstring outlining what these functions do. Currently we're using mkdocstrings and those docstrings can be used in documentation.